pom.xml

```diff
@@ -82,7 +82,6 @@
         <module>seatunnel-core</module>
         <module>seatunnel-transforms</module>
```

Comment:
delete `seatunnel-transforms` ?

Reply:
https://github.com/apache/incubator-seatunnel/pull/3396
https://github.com/apache/incubator-seatunnel/pull/3409
we still have code dependencies, we can do this later